builtin/credential/ldap/backend.go

```diff
@@ -132,6 +132,15 @@ func (b *backend) Login(ctx context.Context, req *logical.Request, username stri
 		return nil, logical.ErrorResponse(err.Error()), nil, nil
 	}
 
+	if cfg.AnonymousGroupSearch {
+		c.Close()
+		c, err = ldapClient.DialLDAP(cfg.ConfigEntry)
+		if err != nil {
+			return nil, logical.ErrorResponse("ldap operation failed"), nil, nil
+		}
+		defer c.Close()
```

Comment:
I think you can drop this line since we already added `c.Close()` to the defer-stack a few lines above.

Reply:
Sure!